server/apiRoutes/event.js

```diff
@@ -125,14 +117,14 @@ router.get('/events', async (req, res, next) => {
 });
 
 //get all attendees for an event
-router.get('/events/:id', async (req, res, next) => {
-  try {
-    const event = await Event.findOne({ where: { id: req.params.id } });
-    const eventAttendees = await event.getUsers();
-    res.send(eventAttendees);
-  } catch (err) {
-    next(err);
-  }
+router.get('/:id/users', async(req, res, next) => {
+    try {
+        const event = await Event.findOne({ where: {id: req.params.id} });
+        const eventAttendees = await event.getUsers();
```

Comment:
Same here as my above thought about attributes

Reply:
actually, didn't end up using this route at all. I'm going to delete it.